pom.xml

```diff
@@ -185,7 +185,7 @@
         <dep.drift.version>1.21</dep.drift.version>
         <dep.duct-tape.version>1.0.8</dep.duct-tape.version>
         <dep.errorprone.version>2.23.0</dep.errorprone.version>
-        <dep.flyway.version>9.22.3</dep.flyway.version>
+        <dep.flyway.version>10.0.0</dep.flyway.version>
```

Comment:
nit: `x.0.0` looks like new bugs on the way.

Reply:
```
Java compatibility
Deprecated Java 8. Java 17 is now required for development
```